components/Footer.tsx

```diff
@@ -1,6 +1,7 @@
 import Image from "next/image";
 import Link from "next/link";
-import ICAF_logo from "../public/icaf-logo-transparent.svg";
+import ICAF_logo_desktop from "../public/icaf-logo-transparent.svg";
+import ICAF_logo_mobile from "../public/icaf-logo-transparent-small.svg";
```

Comment:
Do we need to have two svgs? Perhaps supplying a desired height/width to one svg at the desired breakpoints could work.